plugins/versionpress/src/Utils/AbsoluteUrlReplacer.php

```diff
@@ -100,9 +100,14 @@ private function replaceRecursively($value, $replaceFn)
 
                     $r = new \ReflectionObject($value);
                     $p = $r->getProperties();
-                    foreach ($p as $prop) {
-                        $prop->setAccessible(true);
-                        $prop->setValue($value, $this->replaceRecursively($prop->getValue($value), $replaceFn));
+                    if (get_class($value) != 'wpdb' && get_class($value) != 'mysqli' && get_class($value) != 'mysqli_result') {
+                        foreach ($p as $prop) {
+                            $prop->setAccessible(true);
+                            $newValue = $this->replaceRecursively($prop->getValue($value), $replaceFn);
+                            if (get_class($newValue) != 'wpdb' && get_class($newValue) != 'mysqli' && get_class($newValue) != 'mysqli_result') {
```

Comment:
I am getting some PHP Warnings here, for example `get_class() expects parameter 1 to be object, null given in ...\src\Utils\AbsoluteUrlReplacer.php on line 107`.